src/pages/Subscriptions/SubscriptionsMonthlyGood.vue

```diff
@@ -348,6 +348,12 @@ export default {
 		}
 	},
 	methods: {
+		closeLightbox() {
+			// this will trigger only when the lightbox is closed manually
```

Comment:
Looking at KvLightbox, @lightbox-closed should fire on outside click, esc, or close button.

Reply:
Good catch @ryan-ludwig this is working as intended, but the comment is incorrect. I will update this comment in the next PR